app/[eventSlug]/event.tsx

```diff
@@ -23,9 +23,7 @@ export function EventDisplay(props: {
 }) {
   const { event, days, locations, guests, rsvps } = props;
   const daysForEvent = days.filter(
-    (day) =>
-      !CONSTS.MULTIPLE_EVENTS ||
-      (day["Event name"] && day["Event name"][0] === event.Name)
+    (day) => !CONSTS.MULTIPLE_EVENTS || day.EventName === event.Name
```

Comment:
Are you sure this is the same thing? If yes, I'm a bit confused what the code was doing before.

Reply:
No, it does not. The previous code was just wrong.
Although now that you've pointed it out I'm getting a doubt on which piece was wrong, if it's the type annotation I need to change those.

I'll create another airtable base with multiple events and check

Reply:
I've reverted this change, I can look into it later